.circleci/config.yml

```diff
@@ -552,6 +609,16 @@ workflows:
             branches:
               only:
                 - master
+      - deploy-statviz:
```

Comment:
there's already `deploy-shared` below which should be used (update `requires` field to `build-statviz-staging`)